src/main/java/classes/ToDos.java

```diff
@@ -0,0 +1,16 @@
+package classes;
+
+public class ToDos extends Task {
```

Comment:
Though not explicitly mentioned in the coding standard, perhaps this class should be named `ToDo` instead of `ToDos` (i.e. without the s)? All of the examples for class name given in the standard are singular nouns, and this change would also make the naming of this class consistent with the others in this project (e.g. `Deadline`, `Event`).

Reply:
I agree with you on this. Could I just check with you is the term "ToDo" some sort of reserved word or anything? 
I noticed it highlighted in yellow and everything else aft that turned yellow when i mistyped "ToDos" as "ToDo" when commenting. ( // ...)